Lib/types.py

```diff
@@ -60,7 +60,27 @@ def _m(self): pass
 
 # Provide a PEP 3115 compliant mechanism for class creation
 def new_class(name, bases=(), kwds=None, exec_body=None):
-    """Create a class object dynamically using the appropriate metaclass."""
+    """Create a class object dynamically using the appropriate metaclass.
+
+    Arguments:
+    name -- the new class name
```

Comment:
Please indent the argument list.

Reply:
I tried to do it just like in PEP 257 which doesn't have indentation. But I'll indent next time.